### PR TITLE
Introduce YAML config file

### PR DESCRIPTION
**Pull request by grafnu**

### CODE REVIEW

File: misc/system.yaml [hunk 1]
```diff
@@ -0,0 +1,27 @@
+---
+# Base configuration file for DAQ.
+#
+# This setup will create and test one faux device. To specify a real USB
+# adapter, modify the intf_names entry with the appripriate interface name(s).
+# The startup_cmds line can then be commented out to prevent the (unnecessary)
+# startup of the faux test device.
+
+# Description description for dashboard.
```
Comment (henry54809) [inline misc/system.yaml:9]: this duplicated description is still there

File: proto/system_config.proto [hunk 2]
```diff
@@ -0,0 +1,108 @@
+/*
+ * Specification for DAQ configuration.
+ */
+syntax = "proto3";
+
+/*
+ * DAQ configuration.
+ */
+message ForchConfig {
+  // Description description for dashboard.
```
Comment (henry54809) [inline proto/system_config.proto:10]: same  here.

File: proto/system_config.proto [hunk 3]
```diff
@@ -0,0 +1,108 @@
...
+  // Location of mud files to use for this setup.
+  string mud_files = 26;
+
+  // File to use for runtime device behavior specification.
+  string device_specs = 27;
+
+  // Configuration directory for runtime tests.
+  string test_config = 28;
+
+  // Set port-debounce for flaky connecitons. Zero to disable.
```
Comment (henry54809) [inline proto/system_config.proto:94]: typo

File: misc/system.yaml (hunk 1 above)
Comment (grafnu) [reply misc/system.yaml]: updated

File: proto/system_config.proto (hunk 2 above)
Comment (grafnu) [reply proto/system_config.proto]: updated

File: proto/system_config.proto (hunk 3 above)
Comment (grafnu) [reply proto/system_config.proto]: fixed

**Comment by grafnu** [comment]
Updated PTAL

**Pull request merged**